src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -106,26 +101,24 @@ const Content = ({
             const {
               hasHealthData,
               adoption,
-              sessions_adoption,
+              sessionsAdoption,
               stats,
               crashFreeUsers,
               crashFreeSessions,
               sessionsCrashed,
-              totalUsers,
               totalUsers24h,
-              totalSessions,
+              totalProjectUsers24h,
               totalSessions24h,
+              totalProjectSessions24h,
             } = healthData || {};
             const selectedAdoption =
-              activeDisplay === DisplayOption.CRASH_FREE_SESSIONS &&
-              supportsSessionAdoption
-                ? sessions_adoption
```

Comment:
This is where those errors might come from when an old client talks to the new server.